src/main/java/mate/academy/service/MovieServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.service;
+
+import mate.academy.dao.MovieDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Movie;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    private MovieDao movieDao;
+
+    @Override
+    public Movie add(Movie movie) {
+        return movieDao.add(movie);
+    }
+
+    @Override
+    public Movie get(Long id) {
+        return movieDao.get(id).get();
```

Comment:
use orElseThrowException() instead of get

Reply:
Mentors said me to put get here in previous tasks

Reply:
You will throw Exception in any way here. But if you throw your custom exception with the message you will be able to catch it in exception handler or controller and display to the user. And in the documentation, it is said to use orElseThrow() 
[https://prnt.sc/1364bmu](url)